src/Tribe/Widget/Widget_Interface.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+namespace Tribe\Widget;
+
+use Tribe\Events\Views\V2\View_Interface;/**
+ * Interface Widget_Interface
+ *
+ * @since   TBD
+ *@package Tribe\Widget
+ *
+ */
+interface Widget_Interface {
+
+	/**
+	 * Constructor for V2 Widgets.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $id_base         Optional. Base ID for the widget, lowercase and unique. If left empty,
+	 *                                a portion of the widget's class name will be used. Has to be unique.
+	 * @param string $name            Name for the widget displayed on the configuration page.
+	 * @param array  $widget_options  Optional. Widget options. See wp_register_sidebar_widget() for
```

Comment:
```suggestion
	 * @param array<string,mixed>  $widget_options  Optional. Widget options. See wp_register_sidebar_widget() for
```

Here and elsewhere, define the first level or array content.